doc/changelog.md

```diff
@@ -157,78 +293,72 @@
   [{'expval0': (array(-0.0978434), array(-0.19767681))}, (array(-0.0978434), array(-0.19767681))]
   ```
 
-* Add native support for `qml.PSWAP` and `qml.ISWAP` gates on Amazon Braket devices. Specifically, a circuit like
-
-  ```py
-  import pennylane as qml
-  from catalyst import qjit
+* Support has been added for linear algebra functions which depend on computing the eigenvalues
+  of symmetric matrices, such as `np.sqrt_matrix()`.
+  [(#488)](https://github.com/PennyLaneAI/catalyst/pull/488)
 
-  dev = qml.device("braket.local.qubit", wires=2, shots=100)
+  For example, you can compile `qml.math.sqrt_matrix`:
 
-  @qjit
-  @qml.qnode(dev)
-  def f(x: float):
-      qml.Hadamard(0)
-      qml.PSWAP(x, wires=[0, 1])
-      qml.ISWAP(wires=[1, 0])
-      return qml.probs()
+  ```python
+  @qml.qjit
+  def workflow(A):
+      B = qml.math.sqrt_matrix(A)
+      return B @ A
   ```
 
-  would no longer decompose the `PSWAP` and `ISWAP` gates to `SWAP`s, `CNOT`s and `Hadamard`s. Instead it would just call Braket's native `PSWAP` and `ISWAP` gates at runtime.
-  [(#458)](https://github.com/PennyLaneAI/catalyst/pull/458)
-
-* Add support for the `BlockEncode` operator in Catalyst.
-  [(#483)](https://github.com/PennyLaneAI/catalyst/pull/483)
+  Internally, this involves support for lowering the eigenvectors/values computation lapack method
+  `lapack_dsyevd` via `stablehlo.custom_call`.
 
-* Remove copies of TOML device configuration files for Lightning device in Catalyst.
-  [(#472)](https://github.com/PennyLaneAI/catalyst/pull/472)
+* Additional debugging functions are now available in the `catalyst.debug` directory.
+  [(#529)](https://github.com/PennyLaneAI/catalyst/pull/529)
+  [(#522)](https://github.com/PennyLaneAI/catalyst/pull/531)
 
-* Remove `qextract_p` and `qinst_p` from forced-order primitives.
-  [(#469)](https://github.com/PennyLaneAI/catalyst/pull/469)
+  This includes:
 
-* Update `AllocateQubit` and `AllocateQubits` in `LightningKokkosSimulator` to preserve
-  the current state-vector before qubit re-allocations.
-  [(#479)](https://github.com/PennyLaneAI/catalyst/pull/479)
+  - `compile_from_mlir(ir[, compiler, ...])` to compile a Catalyst function to binary code from the
+    provided MLIR.
 
-* Add support for post-selection in mid-circuit measurements. This is currently supported only on Lightning simulators.
-  [(#491)](https://github.com/PennyLaneAI/catalyst/pull/491)
+  - `filter_static_args(args, static_argnums)` to remove static values from arguments using the
+    provided index list.
```

Comment:
This ...

Reply:
Ah, more utility functions? I'll rephrase

Reply:
As these they need to be removed from this changelog entry which is about the debug functions (which are public).

These two are internal utility functions, they might not even deserve explicit mention beyond was is already mentioned in the refactor entry?

Reply:
done!